servers/cromwell/jobs/__main__.py

```diff
@@ -67,6 +68,10 @@
 try:
     with open(capabilities_path) as f:
         capabilities_config = json.load(f)
+    for settings in capabilities_config['displayFields']:
+        if settings['field'].startswith('labels.'):
```

Comment:
It looks like you can specify `minLength`, `maxLength` and a regex `pattern` for fields in the swagger definition, so you could do that for the labels instead of having to write out these checks: https://swagger.io/docs/specification/data-models/data-types/